### ISSUE TITLE
Can't install Karmada as krew plugin

### ISSUE THREAD

**Issue by BartvdBerkHU** [karmada-io/karmada#1784]
**What happened**:
The Karmada krew plugin installation failed due to an inconsistent checksum.

**What you expected to happen**:
Succesful install of Karmada krew plugin

**How to reproduce it (as minimally and precisely as possible)**:
`kubectl krew install karmada`

**Anything else we need to know?**:
Error:
`Updated the local copy of plugin index.
Installing plugin: karmada
W0512 14:01:19.712336    1468 install.go:164] failed to install plugin "karmada": install failed: failed to unpack into staging dir: failed to unpack the plugin archive: checksum does not match, want: 2711a6b7aa51da6ec263f639559399832f8a6edfef989e8c2d8c65bca191087d, got 255c1f410ff3108c777b5924ec49b7604a3c49ed2284df817a6f4015f22c884f
failed to install some plugins: [karmada]: install failed: failed to unpack into staging dir: failed to unpack the plugin archive: checksum does not match, want: 2711a6b7aa51da6ec263f639559399832f8a6edfef989e8c2d8c65bca191087d, got 255c1f410ff3108c777b5924ec49b7604a3c49ed2284df817a6f4015f22c884f`

**Comment by RainbowMango** [comment]
Very sorry for that. I just realized that it's my fault as I updated the assets manually to fix #1734.

Let me see how to fix it. I guess there are two options:
- Update the checksum recorded in `krew`
- Cut a new release, then `krew` will adopt the newest plugin.

How do you think?

**Comment by RainbowMango** [comment]
Hi @BartvdBerkHU I just updated the checksum and tested it on my side. 

```
# kubectl krew install karmada
Updated the local copy of plugin index.
  New plugins available:
    * accurate
    * advise-policy
    * aws-auth
    * cm
    * cnpg
    * datree
    * dds
    * directpv
    * dtlogin
    * explore
    * htpasswd
    * ice
    * ingress-rule
    * istiolog
    * kadalu
    * karmada
    * kruise
    * ktop
    * kuota-calc
    * kurt
    * lineage
    * linstor
    * log2rbac
    * moco
    * multiforward
    * nsenter
    * print-env
    * pv-migrate
    * realname-diff
    * relay
    * rename-pvc
    * rook-ceph
    * safe
    * secretdata
    * slice
    * stern
    * strace
    * viewnode
    * volsync
    * vpa-recommendation
  Upgrades available for installed plugins:
    * krew v0.4.1 -> v0.4.3
Installing plugin: karmada
Installed plugin: karmada
\
 | Use this plugin:
 | 	kubectl karmada
 | Documentation:
 | 	https://github.com/karmada-io/karmada
/
WARNING: You installed plugin "karmada" from the krew-index plugin repository.
   These plugins are not audited for security by the Krew maintainers.
   Run them at your own risk.

```

Thanks for reporting this.
